server/src/main/java/com/cloud/storage/VolumeApiServiceImpl.java

```diff
@@ -1934,6 +1934,18 @@ private void validateVolumeReadyStateAndHypervisorChecks(VolumeVO volume, long c
         }
     }
 
+    private void setNewIopsLimits(VolumeVO volume, DiskOfferingVO newDiskOffering, Long[] newMinIops, Long[] newMaxIops) {
+        if (newDiskOffering.isCustomizedIops() != null && newDiskOffering.isCustomizedIops()) {
```

Comment:
We can use `org.apache.commons.lang3.BooleanUtils#isTrue` here.

Reply:
or `Boolean.TRUE.equals()`! I am not sure but I think the standard way is good enough.